[bug]: Makefile downloader does not follow redirects.

### Severity

Blocker

### Versions

*

### Components/Modules

sounds

### Operating Environment

*

### Frequency of Occurrence

Constant

### Issue Description

The wget run by the Makefile on building does not follow redirects.

Seems that a recent change to the host of http://downloads.asterisk.org/pub/telephony/sounds/releases/asterisk-core-sounds-en-ulaw-1.6.1.tar.gz triggered this.

### Relevant log output

```shell
root@3d8a862be362:/usr/src/asterisk# make install
CC="cc" CXX="g++" LD="" AR="" RANLIB="" CFLAGS="" LDFLAGS="" make -C menuselect CONFIGURE_SILENT="--silent" makeopts
make[1]: Entering directory '/usr/src/asterisk/menuselect'
make[1]: 'makeopts' is up to date.
make[1]: Leaving directory '/usr/src/asterisk/menuselect'
Installing modules from channels...
Installing modules from pbx...
Installing modules from apps...
Installing modules from codecs...
Installing modules from formats...
Installing modules from cdr...
Installing modules from cel...
Installing modules from bridges...
Installing modules from funcs...
Installing modules from tests...
Installing modules from main...
Installing modules from res...
Installing modules from addons...
/usr/bin/install -c -m 755 contrib/scripts/astversion "/usr/sbin/"
/usr/bin/install -c -m 755 contrib/scripts/astgenkey "/usr/sbin/"
/usr/bin/install -c -m 755 contrib/scripts/autosupport "/usr/sbin/"
./build_tools/install_subst contrib/scripts/safe_asterisk "/usr/sbin/safe_asterisk";
/usr/bin/install -c -m 644 doc/core-*.xml "/var/lib/asterisk/documentation"
/usr/bin/install -c -m 644 doc/appdocsxml.xslt "/var/lib/asterisk/documentation"
/usr/bin/install -c -m 644 doc/appdocsxml.dtd "/var/lib/asterisk/documentation"
/usr/bin/install -c -m 644 doc/asterisk.8 "/usr/share/man/man8"
/usr/bin/install -c -m 644 doc/astdb*.8 "/usr/share/man/man8"
/usr/bin/install -c -m 644 contrib/scripts/astgenkey.8 "/usr/share/man/man8"
/usr/bin/install -c -m 644 contrib/scripts/autosupport.8 "/usr/share/man/man8"
/usr/bin/install -c -m 644 contrib/scripts/safe_asterisk.8 "/usr/share/man/man8"
if [ -f contrib/firmware/iax/iaxy.bin ] ; then \
        /usr/bin/install -c -m 644 contrib/firmware/iax/iaxy.bin "/var/lib/asterisk/firmware/iax/iaxy.bin"; \
fi
CFLAGS="  -I/usr/include/libxml2 -pipe -Wall -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations   -g3 " build_tools/mkpkgconfig "/usr/lib/pkgconfig";
find static-http | while read x; do \
        if test -d $x; then \
                /usr/bin/install -c -m 755 -d "/var/lib/asterisk/$x"; \
        else \
                /usr/bin/install -c -m 644 $x "/var/lib/asterisk/$x" ; \
        fi \
done
/usr/bin/install -c -m 644 doc/core-en_US.xml "/var/lib/asterisk/static-http";
/usr/bin/install -c -m 644 doc/appdocsxml.xslt "/var/lib/asterisk/static-http";
if [ -d doc/tex/asterisk ] ; then \
        /usr/bin/install -c -d "/var/lib/asterisk/static-http/docs" ; \
        for n in doc/tex/asterisk/* ; do \
                /usr/bin/install -c -m 644 $n "/var/lib/asterisk/static-http/docs" ; \
        done \
fi
for x in images/*.jpg; do \
        /usr/bin/install -c -m 644 $x "/var/lib/asterisk/images" ; \
done
make -C sounds install
make[1]: Entering directory '/usr/src/asterisk/sounds'
#################################################################################################################################################################################################### 100.0%
http://downloads.asterisk.org/pub/telephony/sounds/releases/asterisk-core-sounds-en-ulaw-1.6.1.tar.gz

gzip: stdin: not in gzip format
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
make[1]: *** [Makefile:142: .asterisk-core-sounds-en-ulaw-1.6.1] Error 2
make[1]: Leaving directory '/usr/src/asterisk/sounds'
make: *** [Makefile:465: datafiles] Error 2
```


### Asterisk Issue Guidelines

- [X] Yes, I have read the Asterisk Issue Guidelines

--- a/configure
+++ b/configure
@@ -7705,7 +7705,7 @@ if test "x${WGET}" != "x:"; then
   DOWNLOAD_TO_STDOUT="${WGET} -q -O-"
   DOWNLOAD_TIMEOUT='--timeout=$1'
 elif test "x${CURL}" != "x:"; then
-  DOWNLOAD="${CURL} -O --progress-bar -w \"%{url_effective}\n\""
+  DOWNLOAD="${CURL} -L -O --progress-bar -w \"%{url_effective}\n\""
   DOWNLOAD_TO_STDOUT="${CURL} -Ls"
   DOWNLOAD_TIMEOUT='--max-time $(or $2,$1)'
 else

--- a/configure.ac
+++ b/configure.ac
@@ -296,7 +296,7 @@ if test "x${WGET}" != "x:"; then
   DOWNLOAD_TO_STDOUT="${WGET} -q -O-"
   DOWNLOAD_TIMEOUT='--timeout=$1'
 elif test "x${CURL}" != "x:"; then
-  DOWNLOAD="${CURL} -O --progress-bar -w \"%{url_effective}\n\""
+  DOWNLOAD="${CURL} -L -O --progress-bar -w \"%{url_effective}\n\""
   DOWNLOAD_TO_STDOUT="${CURL} -Ls"
   DOWNLOAD_TIMEOUT='--max-time $(or $2,$1)'
 else

